src/main/java/org/thoughtslive/jenkins/plugins/jira/Site.java

```diff
@@ -56,12 +58,13 @@
   private transient JiraService jiraService = null;
 
   @DataBoundConstructor
-  public Site(final String name, final URL url, final String loginType, final int timeout) {
+  public Site(final String name, final URL url, final String loginType, final int timeout, final int readTimeout) {
```

Comment:
To maintain the passivity, can we just declare this new param as @DataBoundSetter similar to others like password, secret or token please? 

Otherwise upon upgrade, this may cause issues to those who has some sort of script to configure this plugin.